api/v1alpha1/status.go

```diff
@@ -0,0 +1,9 @@
+package v1alpha1
+
+// Status defines the observed state of a generic K8up job. It is used for the
+// operator to determine what to do.
+type Status struct {
```

Comment:
I'm just curious: Is there a reason why you've renamed `K8upStatus` to `Status`?

Reply:
Lazyness :) Kubebuilder takes the Type description when the Property is not documented for the CRD generation
```yaml
            description: Status defines the observed state of a generic K8up job. It is used for the operator to determine what to do.
```
If I left it K8upStatus, then it would take also the comment ("description: K8upStatus defines the observed s..."). Now we could leave the Comment mentioning "Status" only, but then any decent IDE/linting system complains about mismatching type and comment, since type comments should begin with the name of the Type AFAIK.

We could override the description when documenting the field like previously:
```go
	// Status this comments overrides the comment of the struct now.
	Status Status     `json:"status,omitempty"`
```
But then I'd have to document every status field again.

do you feel we should change that?

Reply:
No, it's fine by me :)